public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPrevious.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { TestProvider } from 'test/helpers/TestProvider';
+import { getGrafanaContextMock } from 'test/mocks/getGrafanaContextMock';
+
+import { config, reportInteraction } from '@grafana/runtime';
+
+import { ReturnToPrevious, ReturnToPreviousProps } from './ReturnToPrevious';
+
+const mockReturnToPreviousProps: ReturnToPreviousProps = {
+  title: 'Dashboards Page',
+  href: '/dashboards',
+};
+jest.mock('@grafana/runtime', () => {
+  return {
+    ...jest.requireActual('@grafana/runtime'),
+    reportInteraction: jest.fn(),
+  };
+});
+const setup = () => {
+  const grafanaContext = getGrafanaContextMock();
+  grafanaContext.chrome.setReturnToPrevious(mockReturnToPreviousProps);
+  return render(
+    <TestProvider grafanaContext={grafanaContext}>
+      <ReturnToPrevious {...mockReturnToPreviousProps} />
+    </TestProvider>
+  );
+};
+
+describe('ReturnToPrevious', () => {
+  /* We enabled the feature toggle */
+  config.featureToggles.returnToPrevious = true;
```

Comment:
i would either do this in a beforeEach/afterEach like https://github.com/grafana/grafana/blob/main/public/app/core/components/NestedFolderPicker/NestedFolderPicker.test.tsx#L188-L196, or in your mock further up:
```tsx
jest.mock('@grafana/runtime', () => {
  return {
    ...jest.requireActual('@grafana/runtime'),
    reportInteraction: jest.fn(),
    config: {
      ...jest.requireActual('@grafana/runtime').config,
      featureToggles: {
        ...jest.requireActual('@grafana/runtime').config.featureToggles,
        returnToPrevious: true
      }
    }
  };
});
```

Reply:
Doubt: In NestedFolderPicker, the feature toggles are set to the initials, without the nested folder one, once each test is done. In this code we don't do that, right? Should we do it? 🤔

Reply:
ok 🤔 this is not right, the code is not up-to-date. This is how it currently is:

```jsx
describe('ReturnToPrevious', () => {
  beforeEach(() => {
    /* We enabled the feature toggle */
    config.featureToggles.returnToPrevious = true;
  });
  afterEach(() => {
    window.sessionStorage.clear();
    jest.resetAllMocks();
    config.featureToggles.returnToPrevious = false;
  });